satpy/modifiers/atmosphere.py

```diff
@@ -31,7 +31,41 @@
 
 
 class PSPRayleighReflectance(ModifierBase):
-    """Pyspectral-based rayleigh corrector for visible channels."""
+    """Pyspectral-based rayleigh corrector for visible channels.
+
+    It is possible to use ``lim_low``, ``lim_high``
+    and ``strength`` together to reduce rayleigh correction at high solar zenith angle
+    and make the image transition from rayleigh-corrected to partially/none rayleigh-corrected
+    at day/night edge, therefore producing a more natural look. This reduction starts at
+    solar zenith angle of ``lim_low``, and ends in ``lim_high``. It's linearly scaled
+    between these two angles. The ``strength`` controls the amount of the reduction. When
+    the solar zenith angle reaches ``lim_high``, the rayleigh correction will remain
+    ``(1 - strength)`` of its initial strength.
+
+    To use this function in a YAML configuration file:
+
+    .. code-block:: yaml
+
+      rayleigh_corrected_reduced:
+        modifier: !!python/name:satpy.modifiers.PSPRayleighReflectance
+        atmosphere: us-standard
+        aerosol_type: rayleigh_only
+        lim_low: 70
+        lim_high: 95
+        strength: 0.5
+        prerequisites:
+          - name: B03
+            modifiers: [sunz_corrected]
+        optional_prerequisites:
+          - satellite_azimuth_angle
+          - satellite_zenith_angle
+          - solar_azimuth_angle
+          - solar_zenith_angle
+
+    In the case above, rayleigh correction is reduced gradually starting at solar zenith angle 70°.
+    When reaching 95°, the correction will only remain half of its initial strength.
```

Comment:
to make it more clear it should rather say: "..., the correction will only remain half of its initial strength **at lim_high**."

Reply:
And to make things clearer, I'll choose a 0.4/0.6 example instead of this half/half one.